.eslintrc.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
I dropped this to be consistent with the other catbox plugins or something :) It seems as if `lab` also does a fair amount of linting as well.